src/sentry/conf/server.py

```diff
@@ -935,6 +935,8 @@ def create_partitioned_queues(name):
     "organizations:unhandled-issue-flag": False,
     # Enable "owner"/"suggested assignee" features.
     "organizations:workflow-owners": False,
+    # Enable version 2 of reprocessing (completely distinct from v1)
+    "organizations:reprocessing-v2": False,
```

Comment:
Please make sure to deploy the same change in getsentry.

Reply:
@priscilawebdev since we are not using this for all reprocessing features, can we rename it to `reprocessing-v2-ui`?

Reply:
I renamed it to `reprocessing-ui` 😉